ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/recovery/BasicActionExpiryScannerTest.java

```diff
@@ -0,0 +1,129 @@
+/*
+   Copyright The Narayana Authors
+   SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.hp.mwtests.ts.arjuna.recovery;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.time.Instant;
+
+import com.arjuna.ats.arjuna.AtomicAction;
+import com.arjuna.ats.arjuna.common.Uid;
+import com.arjuna.ats.arjuna.common.recoveryPropertyManager;
+import com.arjuna.ats.arjuna.coordinator.ActionStatus;
+import com.arjuna.ats.arjuna.coordinator.ActionType;
+import com.arjuna.ats.arjuna.coordinator.RecordType;
+import com.arjuna.ats.arjuna.coordinator.TwoPhaseOutcome;
+import com.arjuna.ats.arjuna.objectstore.RecoveryStore;
+import com.arjuna.ats.arjuna.objectstore.StateStatus;
+import com.arjuna.ats.arjuna.objectstore.StoreManager;
+import com.arjuna.ats.arjuna.state.OutputObjectState;
+import com.arjuna.ats.internal.arjuna.common.UidHelper;
+import com.arjuna.ats.internal.arjuna.recovery.AdvancedAtomicActionExpiryScanner;
+import com.arjuna.ats.internal.arjuna.recovery.AdvancedAtomicActionPurgeExpiryScanner;
+
+import org.jboss.logging.Logger;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * This test class covers multiple {@code ExpiryScanner} classes.
+ *
+ * @see BasicActionExpiryScanner
```

Comment:
Remark: my IDE shows this as an error because it's an abstract class, googling the fix says to use `{@inheritDoc}`. It's slightly annoying because one's attention is naturally drawn to IDE reported errors.

Reply:
Hmm, I think `{@inheritDoc}` is completely wrong because it's a test class not extending the class BasicActionExpiryScanner. Maybe `@link` is better

Reply:
ok

Reply:
FYI `@link` has the same problem as `@see` unless you fully qualify the class but my IDE seems quite happy to use `{@inheritDoc BasicActionExpiryScanner}`

Reply:
but using @see and importing `import com.arjuna.ats.internal.arjuna.recovery.BasicActionExpiryScanner;` seems like the optimum javadoc.